src/lavatory/commands/purge.py

```diff
@@ -36,15 +37,11 @@
 def purge(ctx, dryrun, policies_path, default, repo, repo_type):
     """Deletes artifacts based on retention policies."""
     LOG.debug('Passed args: %s, %s, %s, %s, %s, %s', ctx, dryrun, policies_path, default, repo, repo_type)
-    artifactory = Artifactory(repo_name=None)
-    before_purge_data = artifactory.list(repo_type=repo_type)
```

Comment:
This variable gets used on line 86. Was this able to even run?

Reply:
Looks like in line 86, it is passed as argument which I replaced in line 44.

Reply:
ahhh ok, my bad

Reply:
ah ok, my bad